pkg/apis/toolchain/v1alpha1/templateupdaterequest_types.go

```diff
@@ -34,22 +42,32 @@ type TemplateUpdateRequestSpec struct {
 // +k8s:openapi-gen=true
 type TemplateUpdateRequestStatus struct {
 	// Conditions is an array of current TemplateUpdateRequest conditions
-	// Supported condition types: ConditionReady
+	// Supported condition types: TemplateUpdateRequestComplete
 	// +optional
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=type
 	Conditions []Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+
+	// SyncIndexes is a map of the sync indexes per cluster before the template refs
+	// were updates in the MasterUserRecord
+	// +optional
+	// +patchMergeKey=type
+	// +patchStrategy=merge
```

Comment:
Don't you need a mapType declaration?
```suggestion
	// +patchStrategy=merge
	// +mapType=atomic
```